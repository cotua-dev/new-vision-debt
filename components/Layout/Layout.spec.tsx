import { render, RenderResult } from '@testing-library/react';
import { Layout } from './index';

describe('Layout Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<Layout/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
