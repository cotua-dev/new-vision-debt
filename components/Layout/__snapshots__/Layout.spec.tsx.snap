// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Component renders 1`] = `
<nav
  class="debt-advisors-navigation"
>
  <a
    aria-label="Home Page Link"
    data-testid="nav-logo-link"
    href="/"
  >
    <picture>
      <source />
      <img
        alt="Debt Advisors Of America Logo"
        class="logo"
        data-testid="regular-logo"
        loading="lazy"
      />
    </picture>
  </a>
  <div
    class="links-wrapper"
  >
    <a
      aria-label="Home Page Link"
      class="link"
      data-testid="home"
      href="/"
    >
      Home
    </a>
    <a
      aria-label="Options Page Link"
      class="link"
      data-testid="options"
      href="/options"
    >
      Options
    </a>
    <a
      aria-label="About Us Page Link"
      class="link"
      data-testid="about-us"
      href="/about-us"
    >
      About Us
    </a>
  </div>
  <button
    aria-label="Mobile Navigation Toggle Button"
    class="mobile-nav-toggle"
    data-testid="open-mobile-navigation-button"
    type="button"
  >
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-bars hamburger"
      data-icon="bars"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 448 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 96C0 78.33 14.33 64 32 64H416C433.7 64 448 78.33 448 96C448 113.7 433.7 128 416 128H32C14.33 128 0 113.7 0 96zM0 256C0 238.3 14.33 224 32 224H416C433.7 224 448 238.3 448 256C448 273.7 433.7 288 416 288H32C14.33 288 0 273.7 0 256zM416 448H32C14.33 448 0 433.7 0 416C0 398.3 14.33 384 32 384H416C433.7 384 448 398.3 448 416C448 433.7 433.7 448 416 448z"
        fill="currentColor"
      />
    </svg>
  </button>
</nav>
`;
